kpi/models/import_export_task.py

```diff
@@ -395,7 +396,150 @@ def export_upload_to(self, filename):
     return posixpath.join(self.user.username, 'exports', filename)
 
 
-class ExportTask(ImportExportTask):
+class ExportBase:
```

Comment:
🤔 
Let me suggest something. `ExportBase` is not really a model but provide some functionality to get export object. Moreover the children class do not inherit from it as their base class but their secondary one.
What about call it a mixin (e.g. `ExportObjectMixin`) and move it into the `mixins` folder?
IMO, it would be make more sense to import a mixin in `renderers.py` too (than a base model class).

Reply:
That's a great idea ❤️